packages/itwinui-css/src/expandable-block/expandable-block.scss

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../utils/index';
+
+@mixin iui-expandable-block {
+  --_iui-expandable-block-expander-icon-fill: var(--iui-color-icon);
+  @include iui-expandable-block-variant;
+  @include iui-expandable-block-size;
+
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  border-radius: var(--iui-border-radius-1);
+
+  &:where(.iui-expanded) {
+    --_iui-expandable-block-expander-icon-transform: rotate(90deg);
+  }
+}
+
+/// @arg $variant Can be one of: default, borderless
+@mixin iui-expandable-block-variant($variant: 'default') {
+  @if $variant == 'default' {
+    --_iui-expandable-block-border: 1px solid var(--iui-color-border);
+    --_iui-expandable-block-background-color: var(--iui-color-background);
+    --_iui-expandable-block-content-padding: var(--iui-size-s);
+    --_iui-expandable-block-header-background-color-disabled: var(--iui-color-background-disabled);
+    --_iui-expandable-block-header-background-color-hover: var(--iui-color-background-hover);
+    --_iui-expandable-block-header-focus-offset: -2px;
+    --_iui-expandable-block-header-focus-thickness: 2px;
+
+    &:where(.iui-expanded) {
+      --_iui-expandable-block-header-bottom-border-radii: 0;
+    }
+  }
+  @if $variant == 'borderless' {
+    --_iui-expandable-block-border: none;
+    --_iui-expandable-block-background-color: transparent;
+    --_iui-expandable-block-content-padding: none;
+    --_iui-expandable-block-header-background-color-disabled: transparent;
+    --_iui-expandable-block-header-background-color-hover: var(--iui-color-background-transparent-hover);
+    --_iui-expandable-block-header-focus-offset: -1px;
+    --_iui-expandable-block-header-focus-thickness: 1px;
+
+    &:where(.iui-expanded) {
+      --_iui-expandable-block-header-bottom-border-radii: inherit;
+    }
+  }
+}
+
+/// @arg $size Can be one of: default, small
+@mixin iui-expandable-block-size($size: 'default') {
+  @if $size == 'default' {
+    --_iui-expandable-block-expander-icon-size: var(--iui-size-m);
+    --_iui-expandable-block-header-gap: var(--iui-size-s);
+    --_iui-expandable-block-header-padding: var(--iui-size-s);
+  }
+  @if $size == 'small' {
+    --_iui-expandable-block-expander-icon-size: var(--iui-size-s);
```

Comment:
Previously, this was done by `@include iui-icon-style('s')` ([L123](https://github.com/iTwin/iTwinUI/blob/main/packages/itwinui-css/src/expandable-block/block.scss#L123)). To confirm, was this a wanted change?

Reply:
This was intentional.  We set the size of expandable block on the parent element.  If we wanted to use `@include iui-icon-style('s')` we would have to target it something like this:
```scss
.iui-expandable-block.iui-small .iui-header .iui-icon { @include iui-icon-style('s') }
```

If I could introduce breaking changes, I could reduce that nesting by giving the icon a more explicit class name like so:
```scss
.iui-expandable-block.iui-small .iui-expandable-block-icon { @include iui-icon-style('s') }
```

This new way, I am able to keep the nesting lower by setting the width / height to a CSS variable `var(--_iui-expandable-block-expander-icon-size)` and then changing that value on the parent.  Something like this:
```scss
.iui-icon {
	width: var(--_iui-expandable-block-expander-icon-size);
	height: var(--_iui-expandable-block-expander-icon-size);
}

.iui-expandable-block.iui-small {
	--_iui-expandable-block-expander-icon-size: 12px;
}
```